modules/message/preview/Clock.tsx

```diff
@@ -14,23 +14,23 @@ const Display = styled.span`
     theme.appearance.display === "cozy" &&
     css`
       margin-left: ${rem(4)};
-
       font-size: ${rem(12)};
       font-weight: 500;
       line-height: ${rem(22)};
       vertical-align: baseline;
+      cursor: default;
     `};
 
   ${({ theme }) =>
     theme.appearance.display === "compact" &&
     css`
       width: ${rem(48)};
       margin-right: ${rem(8)};
-
       font-size: ${rem(11)};
       line-height: ${rem(22)};
       text-align: right;
       text-indent: 0;
+      cursor: default;
```

Comment:
Suggestions don't work here so revert removing the gap (it separated dimensions from text styling) and move the `cursor` property outside of the conditional stuff below height